### PR TITLE
Fix CSS & profile

### PR DESCRIPTION
**Pull request by glu8637**
Fix (some) CSS in shopList page & add profile page

### CODE REVIEW

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
merge conflict

File: package.json
Comment (nonorze-VOXX) [inline package.json:1]: what is this

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: package.json
Comment (glu8637) [reply package.json:1]: idk it appear by itself

Comment (nonorze-VOXX) [reply package.json:1]: delete those npm file in root

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
why /db is changed and backend be changed

**Comment by glu8637** [comment]
> why /db is changed and backend be changed

sry i merged main again to see and resolve conflict

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
remove node module
db should not change

**Pull request closed without merging**